src/app/state.rs

```diff
@@ -81,6 +81,13 @@ pub trait AppState {
         let con = &cc.con;
         Ok(match internal_exec.internal {
             Internal::back => AppStateCmdResult::PopState,
+            Internal::copy_path => {
+                cli_clipboard::set_contents( self.selected_path().to_string_lossy().into_owned() )
+					.map_err( |_| ProgramError::ClipboardError )?
+				;
+
+				AppStateCmdResult::Keep
+            }
```

Comment:
I'm not entirely sure if `AppStateCmdResult::Keep` is the best way. I looked at and liked the behavior or `line_down` where one just has to press enter. One can argue that if I need to copy multiple paths, I'd like to keep the command as is, use arrows to navigate, and hit enter every time I want to copy. 

I initially wanted to clear the command prompt upon invoking this command but couldn't figure how